### PR TITLE
Added endpoint to get all forms without pagination

### PR DESCRIPTION
**Pull request by ivo-ritense**

### CODE REVIEW

File: form/src/main/java/com/ritense/form/service/FormDefinitionService.java [hunk 1]
```diff
@@ -19,14 +19,18 @@
...
+import com.ritense.form.web.rest.dto.FormOption;
+import java.util.List;
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.Pageable;
 import java.util.Optional;
 import java.util.UUID;
 
 public interface FormDefinitionService {
 
-    Page<? extends FormDefinition> getAll(Pageable pageable);
```
Comment (Klaas-Ritense) [inline form/src/main/java/com/ritense/form/service/FormDefinitionService.java:29]: breaking change

**Review by Klaas-Ritense — APPROVED** [review]

File: form/src/main/java/com/ritense/form/service/FormDefinitionService.java (hunk 1 above)
Comment (ivo-ritense) [reply form/src/main/java/com/ritense/form/service/FormDefinitionService.java:29]: Wasn't intended. reverted this change.

File: form/src/main/java/com/ritense/form/repository/FormDefinitionRepository.java [hunk 2]
```diff
@@ -27,6 +28,9 @@
 
 @Repository
 public interface FormDefinitionRepository extends JpaRepository<FormIoFormDefinition, UUID> {
+
+    List<FormIoFormDefinition> findALlByOrderByNameAsc();
```
Comment (Klaas-Ritense) [inline form/src/main/java/com/ritense/form/repository/FormDefinitionRepository.java:32]: ALi?

File: form/src/main/java/com/ritense/form/web/rest/FormManagementResource.java [hunk 3]
```diff
@@ -19,6 +19,7 @@
 import com.ritense.form.domain.FormDefinition;
 import com.ritense.form.domain.request.CreateFormDefinitionRequest;
 import com.ritense.form.domain.request.ModifyFormDefinitionRequest;
+import java.util.List;
```
Comment (Klaas-Ritense) [inline form/src/main/java/com/ritense/form/web/rest/FormManagementResource.java:22]: Not needed

File: form/src/main/java/com/ritense/form/web/rest/impl/FormIoFormManagementResource.java [hunk 4]
```diff
@@ -40,7 +40,7 @@ public FormIoFormManagementResource(FormDefinitionService formDefinitionService)
 
     @Override
     public ResponseEntity<Page<? extends FormDefinition>> getAll(Pageable pageable) {
-        return ResponseEntity.ok(formDefinitionService.getAll(pageable));
+        return ResponseEntity.ok(formDefinitionService.getAllFormOptions(pageable));
```
[the remark this answers is not in the document]
Comment (ivo-ritense) [reply form/src/main/java/com/ritense/form/web/rest/impl/FormIoFormManagementResource.java]: Fixed

File: form/src/main/java/com/ritense/form/web/rest/FormManagementResource.java (hunk 3 above)
Comment (ivo-ritense) [reply form/src/main/java/com/ritense/form/web/rest/FormManagementResource.java:22]: Fixed

File: form/src/main/java/com/ritense/form/repository/FormDefinitionRepository.java (hunk 2 above)
Comment (ivo-ritense) [reply form/src/main/java/com/ritense/form/repository/FormDefinitionRepository.java]: Fixed

**Review by mritense — APPROVED** [review]

**Review by marijnritense — APPROVED** [review]

**Pull request merged**